rskj-core/src/test/java/co/rsk/peg/federation/FederationTestUtils.java

```diff
@@ -69,6 +69,30 @@ public static ErpFederation getErpFederation(NetworkParameters networkParameters
         );
     }
 
+    public static ErpFederation getErpFederation(NetworkParameters networkParameters, List<BtcECKey> fedSigners, long creationBlockNumber) {
```

Comment:
Perhaps we could consider creating a FederationBuilder, what do you think? For another task obviously

Reply:
yes, very good idea. I'll create the task. Thanks :)